internal/testing/alldeps

```diff
@@ -37,6 +37,7 @@ github.com/hashicorp/go-multierror
 github.com/hashicorp/go-retryablehttp
 github.com/hashicorp/go-rootcerts
 github.com/hashicorp/go-sockaddr
+github.com/hashicorp/golang-lru
```

Comment:
To anyone else auditing here: this dependency is released under Mozilla Public License 2.0, which we already have cleared to use from the MySQL driver.